### DISCUSSION TITLE
Nofail mode

### DISCUSSION THREAD

**Discussion by chaostheunderscore** [Leather128/LeathersFunkinEngine#63] [Ideas]
pretty simple, just make an option in the pause menu where nofail stands (and make it automatically on if botplay is on)

**Comment by Leather128** [comment]
uh it's already in there tho???? (no miss)??? or is no fail something different from not missing a note?

**Reply by chaostheunderscore** [reply]
yeah it is nofail is when you cant fail so nofail

**Reply by Leather128** [reply]
oh so you mean like you can't die?

**Reply by chaostheunderscore** [reply]
yea
